### ISSUE TITLE
Ring substituents for some ligands in different orientations, even though same substituent

### ISSUE THREAD

**Issue by hannahbaumann** [openforcefield/protein-ligand-benchmark#104]
I noticed that for some ligands (in the SHP2 and CDK8 sets), ring substituents point into different directions which can potentially lead to sampling problems in the RBFE calculations (if they should point into the same direction and if the ring rotations are not sampled). I understand that the input poses were generated in an automated way and we may not want to refine them by hand afterwards, but I thought I just post my findings here for general awareness:

SHP2:
<img width="597" alt="Screen Shot 2023-11-15 at 3 44 32 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/f6fb38d0-022d-4f5a-a135-c425039e0a19">

CDK8:
<img width="538" alt="Screen Shot 2023-11-15 at 3 50 31 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/8eb51a56-fa98-4690-992a-54817f520fb6">

**Comment by ijpulidos** [comment]
I agree, these poses should be improved. I know people from https://github.com/choderalab/asapdiscovery have successfully used OpenEye docking capabilities to find the poses of compounds, maybe we could use or adapt their pipeline to refine the poses in our database.
